cmd/host/main.go

```diff
@@ -69,6 +75,9 @@ func main() {
 		Forwards: map[string]string{
 			":2222": "192.168.127.2:22",
 		},
+		NAT: map[tcpip.Address]tcpip.Address{
+			tcpip.Address(net.ParseIP(virtualHostIP).To4()): tcpip.Address(net.ParseIP("127.0.0.1").To4()),
```

Comment:
Can you extract the localhost value or use a constant defined by Golang?

Reply:
I used a map[string]string, it's easier to read. I couldn't find a constant for 127.0.0.1 in golang, nor gvisor. Keep it as it is. It's just demo code here.